src/main/java/fi/okm/jod/yksilo/controller/ehdotus/MahdollisuudetController.java

```diff
@@ -195,10 +220,11 @@ public record EhdotusMetadata(
       @Nullable Trendi trendi,
 
       /** Value from 0 to */
-      @Nullable Integer tyollisyysNakyma) {
+      @Nullable Integer tyollisyysNakyma,
+      @NotNull Integer lexicalOrder) {
```

Comment:
Other attributes are in Finnish, so perhaps this should be `aakkosJarjestys` or something.

Reply:
Changed